src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -94,9 +95,10 @@ public void parseCommand_list() throws Exception {
 
     @Test
     public void parseCommand_addMember() throws Exception {
+        String name = "name";
```

Comment:
not sure if it would be nice to have a slightly more semantic name here?
```suggestion
        String name = "Firstname Lastname";
```